Makefile

```diff
@@ -173,6 +173,20 @@ eunit: couch
 	  $(REBAR) -r eunit $(EUNIT_OPTS) apps=$$dir || exit 1; \
 	done
 
+.PHONY: exunit
+# target: exunit - Run ExUnit tests
+exunit: export BUILDDIR = $(shell pwd)
+exunit: export MIX_ENV=test
+exunit: export ERL_LIBS = $(shell pwd)/src
+exunit: export ERL_AFLAGS = -config $(shell pwd)/rel/files/eunit.config
+exunit: export COUCHDB_QUERY_SERVER_JAVASCRIPT = $(shell pwd)/bin/couchjs $(shell pwd)/share/server/main.js
```

Comment:
would it be possible to add a new command where we can specify an app or file to test.

Reply:
Fixed. The syntax is:
```
make exunit tests=src/couch/test/exunit/crud_test.exs,src/fabric/test/exunit/crud_test.exs:line
```

Reply:
awesome. That is useful